snowfall-lib/home/default.nix

```diff
@@ -355,10 +355,12 @@ in {
 
                 home-manager = {
                   users.${user-name} = mkIf config.snowfallorg.users.${user-name}.home.enable ({pkgs, ...}: {
-                    imports = (home-config.imports or []) ++ other-modules ++ [user-module];
+                    imports = (home-config.imports or []) ++ other-modules;
                     config = builtins.removeAttrs home-config ["imports"];
                   });
 
+                  sharedModules = optional config.snowfallorg.users.${user-name}.home.enable user-module;
```

Comment:
This will include the user-specific module for all home-manager users. I don't think we can use this solution since it would break multi-user setups.